specification/recoveryservicessiterecovery/resource-manager/readme.md

```diff
@@ -33,6 +33,11 @@ directive:
       - $.paths
     suppress:
       - UniqueResourcePaths
+
+directive:
+  - from: notRequiredDiscriminatorWithSuppression.json
```

Comment:
The value of `from` should be the file name you're going to suppress. I assume it is service.json in this case.